src/components/StateSvg/StateCircleSvg.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import Circle from './Circle';
 
-const StateCircleSvg = ({
-  state,
-  ratio,
-  actionBackgroundFill,
-  fillColor,
-  hasAction = false,
-}) => {
+const StateCircleSvg = ({ state, ratio, fillColor, hasAction = false }) => {
   return (
     <Circle
       ratio={ratio}
-      actionBackgroundFill={actionBackgroundFill}
+      actionBackgroundFill="#FFFFFF"
```

Comment:
Is there a constant for white we could use?

Reply:
i actually think this entire `actionBackgroundFill` can come out. it looks like a remnant from a `hasAction` prop that doesnt seem like its used anywhere anymore

Reply:
punting this to old-code-cleanup-time